libs/expo/betterangels/src/lib/screens/AddEditClient/PersonalInfo/Dob.tsx

```diff
@@ -21,6 +21,7 @@ export default function Dob() {
         mode="date"
```

Comment:
Hi @Davit-BetterAngels 

I checked this PR on my Android but after I loaded the app through the QR code, I can't select anything to get to the Profile page... Can you check please? Thanks. :)

Reply:
Now, I was able to test with Android. It works as expected. I was able to change the year earlier than 1969 December. :)